cluster-service/Makefile

```diff
@@ -1,57 +1,45 @@
 SHELL = /bin/bash
-AKSCONFIG ?= svc-cluster
-
-CONFIG_PROFILE ?= dev
-include ../dev-infrastructure/configurations/$(CONFIG_PROFILE).mk
-
-CONSUMER_NAME ?= $(shell az aks list --query "[?tags.clusterType == 'mgmt-cluster' && starts_with(resourceGroup, '$(REGIONAL_RESOURCEGROUP)')].resourceGroup" -o tsv)
-KEYVAULT_NAME ?= $(shell az keyvault list --query "[?tags.aroHCPPurpose=='service'].name" -g  ${SVC_KV_RESOURCEGROUP} --output tsv)
-FPA_CERT_NAME ?= firstPartyCert
-AZURE_FIRST_PARTY_APPLICATION_CLIENT_ID ?= "57e54810-3138-4f38-bd3b-29cb33f4c358"
+DEPLOY_ENV ?= personal-dev
+$(shell ../templatize.sh $(DEPLOY_ENV) config.tmpl.mk config.mk)
+include config.mk
 
 deploy:
-	ZONE_RESOURCE_ID=$(shell az network dns zone list -g ${REGIONAL_RESOURCEGROUP} --query "[?zoneType=='Public'].id" -o tsv) && \
-	sed -e "s#ZONE_RESOURCE_ID#$${ZONE_RESOURCE_ID}#g" -e "s/REGION/${REGION}/g" -e "s/CONSUMER_NAME/${CONSUMER_NAME}/g" deploy/mvp-provisioning-shards.yml > deploy/tmp-provisioning-shard.yml
-	ISTO_VERSION=$(shell az aks list --query "[?tags.clusterType == 'svc-cluster' && starts_with(resourceGroup, '${RESOURCEGROUP}')].serviceMeshProfile.istio.revisions[-1]" -o tsv)  && \
+	sed -e "s#ZONE_RESOURCE_ID#${ZONE_RESOURCE_ID}#g" -e "s/REGION/${REGION}/g" -e "s/CONSUMER_NAME/${CONSUMER_NAME}/g" deploy/mvp-provisioning-shards.yml > deploy/tmp-provisioning-shard.yml
```

Comment:
Any reason this is not `yq` or `jq`? Substitutions on structured data with `sed` scare me.

Reply:
this has been on my bucket list for a while. i'm going to use `templatize` directly to generate this

Reply:
```
@../templatize.sh $(DEPLOY_ENV) deploy/provisioning-shards.tmpl.yml deploy/provisioning-shards.yml -e zoneResourceId=$(ZONE_RESOURCE_ID)
```